graphql/resolvers.go

```diff
@@ -2591,9 +2591,22 @@ func (r *taskResolver) CanSetPriority(ctx context.Context, obj *restModel.APITas
 }
 
 func (r *taskResolver) Status(ctx context.Context, obj *restModel.APITask) (string, error) {
+	annotation, err := annotations.FindOneByTaskIdAndExecution(*obj.Id, obj.Execution)
+	if err != nil {
+		return "", InternalServerError.Send(ctx, fmt.Sprintf("error finding task annotation: %s", err.Error()))
+	}
+
+	if annotation != nil && len(annotation.Issues) > 0 {
+		return evergreen.TaskKnownIssue, nil
+
+	}
```

Comment:
We can make some optimizations so we don't this lookup unnecessarily
`DisplayStatus` might already be `KnownIssue` if a task is returned by the `GetTasksByVersion` query so we should check if the status is already Known Issue and if so don't do this query. 

We should also only do this check if a task is in a state where there could be annotations 
eg. Only a failure status. We are already planning on making the same change in the the getTasksByVersion query https://jira.mongodb.org/browse/EVG-15279 

tbh we might even be able to omit this logic and only add the `OriginalStatus` field  and leave this as is. What do you think?